tests/drivers/build_all/sensor/sensors_trigger_own.conf

```diff
@@ -37,6 +37,7 @@ CONFIG_LSM6DSL_TRIGGER_OWN_THREAD=y
 CONFIG_LSM6DSO_TRIGGER_OWN_THREAD=y
 CONFIG_LSM6DSO16IS_TRIGGER_OWN_THREAD=y
 CONFIG_LSM6DSV16X_TRIGGER_OWN_THREAD=y
+CONFIG_MC3419_TRIGGER_OWN_THREAD=y
```

Comment:
Please add this sensor to `sensors_trigger_global.conf` and `sensors_trigger_none.conf` too

Reply:
The driver doesn't have this config. By default, driver is built for global thread mode and had config only for own thread mode.